llama_stack/apis/common/job_types.py

```diff
@@ -3,21 +3,42 @@
 #
 # This source code is licensed under the terms described in the LICENSE file in
 # the root directory of this source tree.
+from datetime import datetime
 from enum import Enum
+from typing import Optional
 
 from pydantic import BaseModel
 
 from llama_stack.schema_utils import json_schema_type
 
 
-@json_schema_type
-class Job(BaseModel):
-    job_id: str
-
-
-@json_schema_type
 class JobStatus(Enum):
     completed = "completed"
     in_progress = "in_progress"
     failed = "failed"
     scheduled = "scheduled"
+    cancelled = "cancelled"
+
+
+class JobType(Enum):
+    batch_inference = "batch_inference"
```

Comment:
Would it make sense to synchronize these with API paths we use for respective APIs? For example, have `eval` and `post-training` (with a dash). I think it would be slightly less error prone / confusing to API users.

Reply:
We have not yet fleshed out "batch_inference" and "post_training" APIs yet, so to limit the scope of this PR, keeping it on scoring and eval only. 

We are also planning on changing `post_training` into `finetuning`.

Reply:
I see. We can adjust this later; maybe add a note here suggesting these values should be synchronized with API paths where possible?